source/fuzz/transformation_replace_id_with_synonym.cpp

```diff
@@ -181,6 +209,43 @@ bool TransformationReplaceIdWithSynonym::UseCanBeReplacedWithSynonym(
 
   return true;
 }
+bool TransformationReplaceIdWithSynonym::IsAgnosticToSignednessOfOperand(
+    SpvOp opcode, uint32_t use_in_operand_index) {
+  switch (opcode) {
+    case SpvOpSNegate:
+    case SpvOpNot:
+    case SpvOpIAdd:
+    case SpvOpISub:
+    case SpvOpIMul:
+    case SpvOpSDiv:
+    case SpvOpSRem:
+    case SpvOpSMod:
+    case SpvOpShiftRightLogical:
+    case SpvOpShiftRightArithmetic:
+    case SpvOpShiftLeftLogical:
```

Comment:
I'm guessing that this is not exhaustive.  For example I'm sure there are a ton of other instructions that - for certain input parameters - are agnostic to the sign of their argument.

Could you open an issue with a TODO(url) comment so that we can track this?

Reply:
For example, OpConvertSToF, OpConvertUToF, OpUConvert, OpSConvert, and I guess many more.

Don't need to do them all in this PR - a follow-on issue would be fine.

Reply:
I opened issue #3582  for this